### DISCUSSION TITLE
How to manage our 20 enterprise seats with Members and Outside Collaborators

### DISCUSSION THREAD

**Discussion by devops-sprouts** [community/community#135523] [Enterprise]
### Select Topic Area

Question

### Body

Hi, We have 20 Github enterprise seats or 30 users, we want to use free licenses for some users or enterprice for some. Can we use free licenses in same github account?

**Comment by queenofcorgis** [comment]
Hi @devops-sprouts 👋🏼 

Thanks for asking this. What type of repositories are you hoping to add the Outside Collaborators to? 

[This doc](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-the-plan-for-your-github-account/about-per-user-pricing#accounts-that-consume-a-license-on-github-enterprise-cloud) shares a breakdown of what accounts would be billed for a license.
